public/mockServiceWorker.js

```diff
@@ -196,20 +186,22 @@ async function resolveMainClient(event) {
 
 async function getResponse(event, client, requestId) {
   const { request } = event
-  const clonedRequest = request.clone()
+
+  // Clone the request because it might've been already used
+  // (i.e. its body has been read and sent to the client).
+  const requestClone = request.clone()
 
   function passthrough() {
-    // Clone the request because it might've been already used
-    // (i.e. its body has been read and sent to the client).
-    const headers = Object.fromEntries(clonedRequest.headers.entries())
+    // Cast the request headers to a new Headers instance
+    // so the headers can be manipulated with.
```

Comment:
```suggestion
    // so the headers can be manipulated
```

Reply:
I do not manage this file, it is generated.